norduniclient/models.py

```diff
@@ -277,6 +277,25 @@ def get_ports(self):
             """
         return core.query_to_list(self.manager, q, handle_id=self.handle_id)
 
+    def add_property(self, property, value):
+        q = """
+            MATCH (n:Node {{handle_id: {{handle_id}}}})
+            SET n.{property} = '{value}'
```

Comment:
I think this will result in that {value} always becomes as string, which is probably not what one expects.

Reply:
One possible solution would be using python six to check if it's a string and format accordingly:
```
        if isinstance(value, six.string_types):
            value = "'{}'".format(value)
        q = """
            MATCH (n:Node {{handle_id: {{handle_id}}}})
            SET n.{property} = {value}
            RETURN n
            """.format(property=property, value=value)
```

Reply:
Yeah, that would work.